WebApi/WebApi.csproj

```diff
@@ -18,6 +18,7 @@
     </PackageReference>
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="6.0.0" />
+    <PackageReference Include="Telegram.Bot" Version="17.0.0" />

```

Comment:
if you added this in Applicaiton, so you can remove this row from here

Reply:
Done